devicon.json

```diff
@@ -5392,6 +5953,9 @@
     },
     {
         "name": "sqlite",
+        "altnames": [
+            "apachesubversion"
```

Comment:
`subversion` and `sqllite`? I know subversion as a version management and sql lite as a database, how does this match?

Reply:
3c5bf839850823623296b0f0a222ba65aff6386c